meetings/conferences/serializers.py

```diff
@@ -1,22 +1,8 @@
-from rest_framework import serializers as ser
-from django_countries.fields import CountryField
+from rest_framework_json_api import serializers
 
 from conferences.models import Conference
 
-class ConferenceSerializer(ser.ModelSerializer):
-    country = CountryField() #get country_dict working later
```

Comment:
Some of the fields are not required, thus we need to specify them since the default is required=True. Examples would be site's url. Also the field country is a package I installed and its installation instruction says to add them in the serializer. We also need the metadata for this field.